### PR TITLE
Create Weekly Backup Policy for Recovery Services Vault to protect VMs

### PR DESCRIPTION
**Pull request by nilaydshah**
This template creates a Weekly Backup Policy that can be used to protect classic and ARM based IaaS VMs.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@nilaydshah__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nilsha). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by nilaydshah** [comment]
I am able to deploy this template successfully (with same parameters file). Not sure why the build is failing.
Can anyone let me know what is the issue here?

File: 101-recoveryservices-weekly-backuppolicy-create/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Create Backup Policy for Recovery Services Vault to protect Azure IaaS VMs
+
```
Comment (singhkay) [inline 101-recoveryservices-weekly-backuppolicy-create/README.md]: @nilaydshah Let's call the folder 101-recovery-services-weekly-backup-policy-create for better readability

**Comment by nilaydshah** [comment]
@singhkay  Updated PR with folder name change

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**